src/imperative/cached_op.h

```diff
@@ -70,6 +71,9 @@ struct CachedOpConfig : public dmlc::Parameter<CachedOpConfig> {
     DMLC_DECLARE_FIELD(is_dynamic)
     .set_default(false)
     .describe("Whether the graph contains dynamic shape operators.");
+    DMLC_DECLARE_FIELD(dynamic_shape_checked)
```

Comment:
Seems it's an internal state of cached_op, I suggest not expose this to end-user.

Reply:
Sure, will do!